libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -142,11 +142,19 @@ func (m *Manager) Apply(pid int) (err error) {
 			}
 			return err
 		}
-		m.Paths[sys.Name()] = p
 
 		if err := sys.Apply(d); err != nil {
+			if os.IsPermission(err) && m.Cgroups.Path == "" {
```

Comment:
We hadn't done this because we couldn't think of a way in which root could get a permission error. Maybe from a Mandatory Access Control?